src/deck.js

```diff
@@ -38,16 +40,22 @@ import { postRichObjectToConversation } from './services/messagesService.js'
 	 */
 	async function postCardToRoom(card, token) {
 		try {
-			const response = await postRichObjectToConversation(token, {
-				objectType: 'deck-card',
-				objectId: card.id,
-				metaData: JSON.stringify(card),
-			})
-			const messageId = response.data.ocs.data.id
+			const [responsePostCard, responseGetConversation] = await Promise.allSettled([
+				postRichObjectToConversation(token, {
+					objectType: 'deck-card',
+					objectId: card.id,
+					metaData: JSON.stringify(card),
+				}),
+				fetchConversation(token),
+			])
+
+			const messageId = responsePostCard.value.data.ocs.data.id
+			const conversation = responseGetConversation.value.data.ocs.data.displayName
 			const targetUrl = generateUrl('/call/{token}#message_{messageId}', { token, messageId })
-			showSuccess(t('spreed', 'Deck card has been posted to the selected <a href="{link}">conversation</a>', {
-				link: targetUrl,
-			}), {
+
+			showSuccess(escapeHtml(t('spreed', 'Deck card has been posted to {conversation}'))
+				.replace(/\{conversation}/g, `<a target="_blank" class="external" href="${targetUrl}">${conversation} ↗</a>`),
```

Comment:
The translation should not need `escapeHtml(`. However the conversation name should have it as that is user input.

Reply:
Fixed